### PR TITLE
docs(menu): change the text color when reaching the white background

### PR DESCRIPTION
**Pull request by vvo**
# Before
before

# After
after

### CODE REVIEW

**Comment by pixelastic** [comment]
I like the idea, this is smoother that way. I would use a lighter color than the black, though. Maybe the text color of the white part?

**Comment by vvo** [comment]
> I like the idea, this is smoother that way. I would use a lighter color than the black, though. Maybe the text color of the white part?

Yes I still find the color too strong also, @Shipow any advice? We could maybe use the last gradient tone/color :)

**Comment by bobylito** [comment]
Wooosh :D Great idea :)

**Comment by bobylito** [comment]
I think we can merge, the color can be the subject of another fix!

File: docs/js/home.js [hunk 1]
```diff
@@ -216,3 +233,38 @@ Star.prototype.draw = function() {
   context.fill();
   context.restore();
 };
+
+// http://stackoverflow.com/questions/14012766/detecting-whether-two-divs-overlap
+function overlap($div1, $div2) {
```
Comment (bobylito) [inline docs/js/home.js]: DOM AABB :D

File: docs/js/home.js [hunk 2]
```diff
@@ -216,3 +233,38 @@ Star.prototype.draw = function() {
...
+  var rect1 = $div1[0].getBoundingClientRect();
+  var rect2 = $div2[0].getBoundingClientRect();
+
+  return !(rect1.right < rect2.left ||
+    rect1.left > rect2.right ||
+    rect1.bottom < rect2.top + 80 ||
```
Comment (bobylito) [inline docs/js/home.js]: Why the magic 80 value?

Comment (vvo) [inline docs/js/home.js]: Sorry this is a top offset to have it exactly change color on the right stop (middle of the svg path), will refactor this later on

**Pull request merged**